packages/oid4vci-holder/src/machine/oid4vciMachine.ts

```diff
@@ -586,6 +586,9 @@ export class OID4VCIMachine {
     }
     if (opts?.requireCustomNavigationHook !== true) {
       interpreter.onTransition((snapshot: OID4VCIMachineState): void => {
+        if (opts?.stateNavigationListener === undefined) {
```

Comment:
Why? Especially since you made the stateNavigationListener optional as well.

Reply:
I don't remember why we did that. Removed in next commit